dotcom-rendering/src/web/components/ShadyPie.tsx

```diff
@@ -10,17 +13,45 @@ params.set(
 );
 params.set('INTCMP', 'shady_pie_open_2019');
 
-export const ShadyPie = () => {
-	return (
-		<a
-			href={`https://support.theguardian.com/uk/subscribe/digital?${params.toString()}`}
-			tabIndex={-1}
-		>
-			<img
-				src="https://uploads.guim.co.uk/2020/10/02/Digisubs_MPU_c1_my_opt.png"
-				width="300"
-				alt=""
+export const ShadyPie = ({
+	sectionName,
+	editionId,
+}: {
+	sectionName: string;
+	editionId: EditionId;
+}) => {
+	let pieFilling;
+	if (sectionName == 'lifestyle' && editionId == 'UK') {
+		pieFilling = (
+			<LabsShadyPie
+				articleTitle="It’s all about the berries: meet one of the farmers growing blackcurrants for Ribena"
+				articleImageLink="https://i.guim.co.uk/img/media/f98f93c22a563b3107394d64e8410a062172de7d/0_37_2122_1273/master/2122.jpg?width=620&quality=45&fit=max&dpr=2&s=9e01623babd043c1a2fbaec9db29fca2"
```

Comment:
Can we reduce this width value from `width=620`? That's pretty big and it looks like we only ever need a smaller size so we can keep the overall page size smaller by adjusting this url